src/event/event_config.h

```diff
@@ -146,9 +152,15 @@
 
 #	define DISPATCH_HAVE_TIMER_QOS 0
 #	define DISPATCH_HAVE_TIMER_COALESCING 0
-#	define KEVENT_FLAG_IMMEDIATE 0x001
 #endif // !DISPATCH_EVENT_BACKEND_KEVENT
 
+// These flags are used by dispatch generic code and
+// translated back by the various backends to similar semantics
+// hence must be defined even on non Darwin platforms
+#ifndef KEVENT_FLAG_IMMEDIATE
+# define KEVENT_FLAG_IMMEDIATE 0x001
```

Comment:
remove the leading space

Reply:
All other #defines are with the leading space on this file... should I still remove them?

Reply:
the big hunk of configuration has a lot of nesting so it is pseudo indented for readability, but once it's at top level we don't use leading indent.

ALso I think it is tab-indented for the big fat one at the top not space ;)

Reply:
@MadCoder changed to a tab AND fixed my tab size... now I can see them 😝 

I kept a tab for consistency with the previous defines.